src/Form/Checkbox/Checkbox.component.tsx

```diff
@@ -0,0 +1,123 @@
+// REACT
+import * as React from 'react';
+const { forwardRef } = React;
+
+// VENDOR
+import styled, { css } from '@xstyled/styled-components';
+import { compose } from '@xstyled/system';
+import classnames from 'classnames';
+
+// ANCHOR
+import { Check } from '../..';
```

Comment:
Does this give you a circular reference error on build?

Reply:
Woops